edgexfoundry-holding/device-onvif-camera
# feat: add GetCapabilities and GetEndpointReference

- Adds general `GetCapabilities`
- Adds `GetServiceCapabilities` for each service
- Adds `GetEndpointReference` command

Signed-off-by: Anthony Casagrande <anthony.j.casagrande@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions


See updated postman collection for how to call the commands.

> _**MAKE SURE YOU ARE USING THE LATEST DEVICE PROFILES PROVIDED HERE**_
The easiest way of ensuring this is to run a `make clean` and start fresh on the `edgex-compose` project.

## New Dependency Instructions (If applicable)
